packages/stripe/lib/src/widgets/apple_pay_button.dart

```diff
@@ -17,6 +17,7 @@ class ApplePayButton extends StatelessWidget {
     Key? key,
     this.style = ApplePayButtonStyle.black,
     this.type = ApplePayButtonType.plain,
+    this.cornerRadius = 4.0,
```

Comment:
personally I would set it to 0.0 since I think it will break the UI for everyone that upgrades the library.

Reply:
The default value is 4.0 from the doc :

https://developer.apple.com/documentation/passkit/pkpaymentbutton/2999416-cornerradius